azure_pipeline-global.yaml

```diff
@@ -39,6 +39,8 @@ parameters:
       storage_account_rg: 'core-infra-sbox-rg'
       storage_account_name: 'cftappssbox'
       dependsOn: 'Precheck'
+      files_list: 'components/global modules/azure-landing-zone environments/sbox/sbox.tfvars environments/variables.tf'
+      override: false
```

Comment:
Can we rename this var to be more obvious ? something like alwaysApply

Reply:
or we could change it to say, if there is no file_list- apply it always.  so that we dont need a new var

Reply:
I can change to alwaysApply , regarding the check for no file_list - possible but if a user wants to apply when a file in that list hasn't changed.

will keep the true/false flag and ammend to alwaysApply